### ISSUE TITLE
Can you remove apperance.js or make it optional/toggle

### ISSUE THREAD

**Issue by Amorano** [jags111/efficiency-nodes-comfyui#199]
I am not trying to be an ass, but I don't like being forced into things when I don't need to be forced. 

I use color as a workflow indicator, its why I wrote all the colorization stuff in my extension.

I would like the nodes to use the colors I have selected using said extension, but the appearance.js file forces it, every time.

I can remove it local, not a problem -- is a problem for other people doing the same thing.

If it was an optional toggle?

The only ones that allow for this currently are the misc nodes:
![image](https://github.com/jags111/efficiency-nodes-comfyui/assets/225178/e3a67403-8675-4d77-9d3e-fa3cc045b836)

**Comment by jags111** [comment]
yes this code was there earlier and will try and strip it if possible

**Comment by lum3on** [comment]
@jags111 I need some advice for changing the appearance on a custom node I am building myself. what is the correct syntax for change?

would this be enough in appearance.js?

`import { app } from "../../scripts/app.js";

app.registerExtension({
    name: "**registered_name**.appearance", 
    async nodeCreated(node) {
        if (node.comfyClass === "**pythonscript_name**") { 
            // Apply styling
            node.color = "#16727c";
            node.bgcolor = "#4F0074";
        }
    }
});`

I registered my node already but it does not change the node, therefore I ask
